data/autoyast_sle15/autoyast_disk_as_pv.xml

```diff
@@ -64,10 +64,28 @@
         </yesno_messages>
     </report>
     <partitioning config:type="list">
+        <drive>
+            <device>/dev/sda</device>
+            <disklabel>gpt</disklabel>
+            <initialize config:type="boolean">true</initialize>
+            <partitions config:type="list">
+                <partition>
+                    <create config:type="boolean">true</create>
+                    <format config:type="boolean">false</format>
+                    <size>8Mib</size>
+                </partition>
+                <partition>
+                    <format config:type="boolean">true</format>
+                    <create config:type="boolean">false</create>
+                    <mount>/boot</mount>
+                    <size>256M</size>
+                </partition>
+            </partitions>
+        </drive>
         <drive>
             <type config:type="symbol">CT_DISK</type>
             <use>all</use>
-            <device>/dev/sda</device>
+            <device>/dev/sdb</device>
```

Comment:
if understand problem correctly it is something similar to what I have for wicked testing and NIC name in config files. After all I came up to approach where in config file I have some anchor which I replace with sed in the run-time when I really know what devices I have. Maybe you should try the same ?

Reply:
Not really, I had to use 3 disks instead of 2. 1 with bios boot and /boot partitions and 2 partitionless for / and LVM.